examples/speaker_tasks/diarization/conf/inference/diar_infer_general.yaml

```diff
@@ -10,6 +10,9 @@ num_workers: 1
 sample_rate: 16000
 batch_size: 64
 device: null # can specify a specific device, i.e: cuda:1 (default cuda if cuda available, else cpu)
+map_location: null
+verbose: True # enable additional logging
+evaluate: True # enable evaluation (if RTTM files are passed in, else ignore)
```

Comment:
Is this parameter required? We are evaluating based on if RTTM files are passed or not

Reply:
Thanks removed this and pass the verbose flag to the logging warning.